bittensor/__init__.py

```diff
@@ -16,16 +16,29 @@
 # DEALINGS IN THE SOFTWARE.
 
 from rich.console import Console
+from rich.traceback import install
 from prometheus_client import Info
 
+import nest_asyncio
+nest_asyncio.apply()
+
 # Bittensor code and protocol version.
-__version__ = '3.4.1'
+__version__ = '3.4.2'
```

Comment:
remember also to update this version: https://github.com/opentensor/bittensor/blob/Release_3.4.2/VERSION

Next times you can use the script: https://github.com/opentensor/bittensor/blob/Release_3.4.2/scripts/release/update_version.sh